server/src/main/kotlin/com/kamelia/jellyfish/rest/user/Service.kt

```diff
@@ -120,7 +135,9 @@ private suspend fun checkEmail(email: String?, toEdit: User? = null) =
                 } else {
                     QueryResult.forbidden("errors.users.email.already_exists")
                 }
-            }
+            } ?: if ("@" !in email) {
+                QueryResult.forbidden("errors.users.email.invalid")
+            } else null
```

Comment:
No, this shan't be done ; mail verification is way too complicated, and we will have to verify mails by sending an email anyway. The only reliable rule is that it always contains an @
Worse than that, some mail servers don't enforce certain rules either, so that would prevent a user from said mail server to use their address.